### ISSUE TITLE
Workflow change: using content, next-release, and feature branches

### ISSUE THREAD

**Issue by therationalpi** [Backdash/MonikaModDev#100]
Just wanted to share our new workflow to make pushing out major and minor releases a little smoother.

Up until now, we've been using one major branch to collect all updates. This has made it difficult to push out minor updates with just bugfixes, script edits, and new content, because any features need to be stable before we can release.

We will now be using three main branches, depending on what you're doing:

`content` is for any dialogue changes (including typos), new topics/greetings, and updates to image assets.

`next-release` is for the next major release, like version 0.6.0 from 0.5.0. This will basically be the most unstable branch, but the one with the most active work.

*feature branches* are experimental branches for developing new features. For example, the branch for building the loading from ddlc was called `load-ddlc`. *Try your best to keep work on this in bits and pieces, and merge into `next-release` whenever you are at a fairly stable point* so we can find feature conflicts quickly. Fight the urge to hold onto the feature branch until its perfect.

`master` will basically be the current version and/or the next release candidate. It's still the place to submit bug fixes, and we'll merge in content before minor releases.

Hopefully this will help make things smoother going forward. If you have any suggestions on this workflow, please let me know!

**Comment by therationalpi** [comment]
Closing since I think everyone has had a chance to read this.
